pinery-lims/src/main/java/ca/on/oicr/pinery/lims/DefaultLaneProvenance.java

```diff
@@ -67,6 +67,9 @@ public String getSequencerRunName() {
         if (sequencerRun != null && sequencerRun.getRunBasesMask() != null && !sequencerRun.getRunBasesMask().isEmpty()) {
         	attrs.put("run_bases_mask", sequencerRun.getRunBasesMask());
         }
+        if (sequencerRun != null && sequencerRun.getSequencingParameters() != null && !sequencerRun.getRunBasesMask().isEmpty()) {
+        	attrs.put("sequencing_parameters", sequencerRun.getSequencingParameters());
```

Comment:
The current version of lane provenance should not be modified. If this attribute is required in provenance, it should go in a new provenance version after #81 is merged. If this attribute is for reporting only and not needed for analysis, it should not be added to provenance at all

Reply:
Fixed